src/constants/styles/colors.ts

```diff
@@ -1,4 +1,12 @@
-import { AppColorMode } from '@/state/configs';
+import type { ThemeName } from 'public/tradingview/charting_library';
+
+import { AppColorMode, AppTheme } from '@/state/configs';
+
+export const THEME_NAMES: Record<AppTheme, ThemeName> = {
+  [AppTheme.Classic]: 'Classic',
```

Comment:
ideally wanted to map this to `[AppTheme.Classic]: ''` but it looks like the type `ThemeName` (coming from `charting_library.d.ts`) is being autogenerated - and I verified that calling `changeTheme('Classic')` (or initializing `theme`) does the same thing as `changeTheme('')` so I think it's fine to leave like this